<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinishedWindow</class>
 <widget class="QDialog" name="FinishedWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">                  QPushButton
                  {
                  background: #890709;
                  text-align: center;
                  color: #FFFFFF;
                  }
                  QPushButton::hover{
                  background: #890709;
                  border: 1px solid #7B59FF;
                  }

                  QPushButton::pressed{
                  background: #890709;
                  }</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableCompare_">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <column>
      <property name="text">
       <string>Вопрос</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ответ</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblResult_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineName_">
     <property name="placeholderText">
      <string>Введите ваше имя...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnSave_">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
